evm_loader/test_transaction.py

```diff
@@ -310,7 +315,7 @@ def test_04_success_tx_send_iteratively_by_3_instructions_in_one_transaction(sel
         self.assertLess(data[1], 0xd0)  # less 0xd0 - success
         self.assertEqual(int().from_bytes(data[2:10], 'little'), 24301)  # used_gas
 
-    # @unittest.skip("a.i.")
+    @unittest.skip("works only in 5 instructions but cant be serialized")
```

Comment:
You can increase `step_count` to finalize this transaction into 4 iterations.

Reply:
fixed